eligibility_api/client.py

```diff
@@ -193,7 +191,10 @@ def _tokenize_response(self, response):
     def _auth_headers(self, token):
         """Create headers for the request with the token and verifier API keys"""
         headers = dict(Authorization=f"Bearer {token}")
-        headers[self.api_auth_header] = self.api_auth_key
+
+        for key, value in self.headers.items():
```

Comment:
I like this, the only thing I would say is maybe filter out `Authorization`? Since we don't want anyone to be able to override that header (since it is created by this function)

Reply:
Thanks for catching that. Would we want to silently filter it out or raise an exception when someone's trying to set an "Authorization" header?

Reply:
Hmm probably your idea - raise an Exception. Then we can write a test for that case 😉

Reply:
Could do that in the `__init__` even, to fail earlier.